src/components/achievements/Stat.astro

```diff
@@ -25,17 +26,22 @@ const repoCounts = Object.entries(stat).sort(([, a], [, b]) => b - a);
     <H class="heading-5 text-white">{type[0].toUpperCase() + type.slice(1)}</H>
     <p class="text-[2rem] text-white font-obviously-wide font-semibold leading-none">{count}</p>
   </div>
-  <ul class="text-[13px] text-white leading-tight">
+  <ul class="text-[13px] leading-tight">
     {
       repoCounts.map(([repo, repoCount]) => {
         const percentage = (repoCount / maxCount) * 100;
         return (
           <li class="grid grid-cols-2 items-center">
-            <span
-              class="text-right py-0.5 pr-2 border-r border-neutral-500 break-words"
+            <a
+              href={`https://github.com/withastro/${repo}/${
```

Comment:
What do you think of moving the logic figuring out the URL up to the frontmatter where it's likely to be a bit more readable? If the ternary is a bit much, it's also easier to break things up there if you need to.

Reply:
Moved the query to frontmatter in https://github.com/delucis/astro-badge/pull/27/commits/1bc3060d8b67bdeb602b4f080d8e46f4547b890e